cmd/influxd/launcher/launcher.go

```diff
@@ -986,7 +986,7 @@ func (m *Launcher) openMetaStores(ctx context.Context, opts *InfluxdOpts) (strin
 
 		// If a sqlite-path is not specified, store sqlite db in the same directory as bolt with the default filename.
 		if opts.SqLitePath == "" {
-			opts.SqLitePath = filepath.Dir(opts.BoltPath) + "/" + sqlite.DefaultFilename
+			opts.SqLitePath = filepath.Join(filepath.Dir(opts.BoltPath), sqlite.DefaultFilename)
```

Comment:
Found this while working on the similar logic in the `downgrade` command. Seemed bad to create a path like this for cross-platform stuff.